NVActivityIndicatorViewDemo/ViewController.swift

```diff
@@ -8,14 +8,9 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, NVActivityIndicatorViewable {
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        self.view.backgroundColor = UIColor(red: CGFloat(237 / 255.0), green: CGFloat(85 / 255.0), blue: CGFloat(101 / 255.0), alpha: 1)
-        
-        let activityTypes: [NVActivityIndicatorType] = [
+    let activityTypes: [NVActivityIndicatorType] = [
```

Comment:
I moved the scope from `viewDidLoad` to the view controller level to simply reference it in the button tap.